skl2onnx/common/_topology.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, raw_name, onnx_name, scope, type=None):
         :param type: A type object defined in .common.data_types.py;
                      e.g., FloatTensorType
         """
-        if not isinstance(raw_name, str) or '(' in raw_name:
+        if not isinstance(raw_name, str):
             raise TypeError(
                 "raw_name must be a string not '%s'." % raw_name.__class__)
```

Comment:
Could you update the error message to reflect that it be raised if the variable name contains a parenthesis?